src/components/ApplicationInfo.tsx

```
@@ -1,40 +1,97 @@
 import React from "react";
-import { VStack } from "@chakra-ui/react";
-import { Table } from "ka-table"; // Importing ka-table directly
-import { DataType } from "ka-table/enums";
+import { Table, DataType } from "ka-table";
+import "ka-table/style.css";
+
 interface ApplicationInfoProps {
-  details: { [key: string]: string | number | boolean | null | undefined };
+  details: { [key: string]: any };
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)